### PR TITLE
 Add geckodriver for Firefox 64, update version

### PR DESCRIPTION
**Pull request by sabgit**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "browser-driver-installer",
-    "version": "1.0.12",
+    "version": "1.0.13",
```
Comment (sabgit) [inline package-lock.json:3]: do we need to update this file?

Comment (cdagli) [reply package-lock.json:3]: No it is generated, if it changes you can commit the changes

Comment (sabgit) [reply package-lock.json:3]: how is it generated, you mean when we install the npm package? I am thinking we might not even need this file here, no?

Comment (cdagli) [reply package-lock.json:3]: It is generated when you install this package using `npm run install`

Comment (sabgit) [reply package-lock.json:3]: then I guess it is ok to remove the file from the repo?

Comment (cdagli) [reply package-lock.json:3]: No we should keep it, it has some benefits: https://docs.npmjs.com/files/package-lock.json#description

**Review by thehcma — APPROVED** [review]

**Review by thehcma — APPROVED** [review]

**Pull request merged**